_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: single
+title: "Big Changes for pyOpenSci - an update from the new executive director, leah wasser, what's next?"
```

Comment:
title: "Big changes for pyOpenSci: an update from the new executive director, Leah Wasser"

Reply:
(suggest shortening and using a colon in the title)

Reply:
done!